docs/source/building-supergraphs/router.mdx

```diff
@@ -33,426 +33,4 @@ Regardless of which graph router library you start with, you can swap to the oth
 
 </ExpansionPanel>
 
-**If you'll use the Apollo Router,** get started with the resources linked above. The remainder of this article covers using Apollo Server with `@apollo/gateway`.
```

Comment:
In this article, as a sub-list-item for the **Apollo Server** item on line 24, can we add a link to the new location for the Apollo Gateway article?